dotnet/src/SemanticKernel.Core/Extensions/FunctionViewExtensions.cs

```diff
@@ -32,21 +32,14 @@ public static JsonSchemaFunctionManual ToJsonSchemaManual(this FunctionView func
         var requiredProperties = new List<string>();
         foreach (var parameter in function.Parameters)
         {
-            if (parameter.ParameterType != null)
+            var schema = parameter.Schema ?? jsonSchemaDelegate(parameter.ParameterType!, parameter.Description ?? string.Empty);
```

Comment:
How do we know that `parameter.ParameterType` is not null here?

Reply:
Fixed